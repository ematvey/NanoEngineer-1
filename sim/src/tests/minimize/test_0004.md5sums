stdout 5B9D7158E91BDB040018D9A6F14A2DEA
stderr D41D8CD98F00B204E9800998ECF8427E
