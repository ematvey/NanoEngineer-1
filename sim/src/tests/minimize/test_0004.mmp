mmpformat 050130 required; 050217 optional
kelvin 300
group (Data)
csys (HomeView) (1.000000, 0.000000, 0.000000, 0.000000) (6.905391) (0.000000, 0.000000, 0.000000) (1.000000)
csys (LastView) (-0.666349, 0.131054, 0.730464, -0.072288) (6.430436) (0.000000, 0.000000, 0.000000) (1.000000)
datum (XY) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 0.000000, 1.000000) (0.000000, 1.000000, 0.000000)
datum (YZ) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (1.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000)
datum (ZX) (0, 0, 255) plane (0.000000, 0.000000, 0.000000) (0.000000, 1.000000, 0.000000) (0.000000, 1.000000, 0.000000)
egroup (Data)
group (AllInOne)
mol (Chunk-Si1) def
atom 1 (14) (-1242, -1372, 301) def
atom 2 (14) (1087, -1426, 237) def
bond1 1
atom 3 (14) (-1910, 861, 302) def
bond1 1
atom 4 (14) (1858, 772, 191) def
bond1 2
atom 5 (14) (5, 2186, 236) def
bond1 3 4
atom 6 (8) (73, 3245, 1715) def
bond1 5
atom 7 (0) (160, 2828, 2220) def
bond1 6
atom 8 (8) (2908, 1087, 1644) def
bond1 4
atom 9 (0) (2526, 949, 2165) def
bond1 8
atom 10 (8) (1724, -2272, 1718) def
bond1 2
atom 11 (0) (1527, -1898, 2226) def
bond1 10
atom 12 (8) (-1837, -2194, 1812) def
bond1 1
atom 13 (0) (-1458, -1955, 2297) def
bond1 12
atom 14 (8) (-2856, 1220, 1815) def
bond1 3
atom 15 (0) (-2432, 1118, 2310) def
bond1 14
atom 16 (8) (-13, 3229, -1265) def
bond1 5
atom 17 (8) (-2947, 1197, -1166) def
bond1 3
atom 18 (8) (-1918, -2216, -1162) def
bond1 1
atom 19 (8) (1642, -2306, -1255) def
bond1 2
atom 20 (8) (2820, 1059, -1337) def
bond1 4
atom 21 (6) (-2089, 962, -2287) def
bond1 17
atom 22 (0) (-2453, 1162, -2935) def
bond1 21
atom 23 (6) (51, 2308, -2357) def
bond1 16 21
atom 24 (0) (86, 2698, -3020) def
bond1 23
atom 25 (6) (-1423, -1473, -2279) def
bond1 18 21
atom 26 (0) (-1743, -1739, -2927) def
bond1 25
atom 27 (6) (1048, -1597, -2345) def
bond1 19 25
atom 28 (0) (1237, -1940, -3008) def
bond1 27
atom 29 (6) (1898, 823, -2405) def
bond1 20 27 23
atom 30 (0) (2242, 970, -3078) def
bond1 29
rmotor (Rotary Motor.2) (112, 0, 0) 0.00 0.00 (-40, 204, 253) (-28, -9, -999) 4.00 0.50 0.20
shaft 1 5 2 3 4
mol (AllInOne-blankPdb.pdb) def
egroup (AllInOne)
end1
group (Clipboard)
egroup (Clipboard)
end molecular machine part AllInOne
