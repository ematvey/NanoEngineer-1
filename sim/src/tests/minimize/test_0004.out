======= test_0004.test =======
TYPE fail
PROGRAM simulator -f600 -m -x test_0004.mmp
